roles/openshift_control_plane/templates/recycler_pod.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "v1"
+kind: "Pod"
+metadata:
+  name: "recyler-pod-"
+  namespace: "openshift-infra"
+spec:
+  activeDeadlineSeconds: 60
+  restartPolicy: "Never"
+  serviceAccountName: "pv-recycler-controller"
+  containers:
+    -
+      name: "recyler-container"
+      image: "docker.io/openshift/origin-recycler:v3.11"
```

Comment:
This should refer to an imagestream, which uses `oreg_url` and has version templated